### PR TITLE
use media_files.result()

### PR DESCRIPTION
**Pull request by TeslaH2O**
media_files is a Future object. Thus, using a for loop to filter files into a list doe snot work.  media_files.result() should be used

### CODE REVIEW

**Comment by MichaIng** [comment]
Many thanks for your contribution.

Looks like the regression was introduced here: https://github.com/motioneye-project/motioneye/commit/ec14dd6

**Comment by MichaIng** [comment]
Pre-commit black wants to enforce the max line length. The Ubuntu build error is unrelated, looks like this particular GitHub actions runner has a wrong EFI target set in GRUB config.

**Review by zagrim — APPROVED** [review]
:+1:

File: motioneye/sendmail.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def on_media_files(media_files):
 
             # filter out non-recent media files
             media_files = [
-                m for m in media_files if abs(m['timestamp'] - timestamp) < timespan
+                m for m in media_files.result() if abs(m['timestamp'] - timestamp) < timespan
```
Comment (MichaIng) [inline motioneye/sendmail.py:93]: Apply pre-commit black format:
```suggestion
                m
                for m in media_files.result()
                if abs(m['timestamp'] - timestamp) < timespan
```

**Review by MichaIng — APPROVED** [review]

**Pull request merged**